R/utils.R

```diff
@@ -327,6 +327,29 @@ h_factor_ref <- function(x, ref, var_name = vname(x)) {
   factor(x, levels = h_default_value(levels(ref), sort(uni_ref)))
 }
 
+#' Convert Character to Factor Following Reference `MMRM` Fit.
+#'
+#' @param object (`mmrm_tmb`)\cr the fitted MMRM object.
+#' @param data (`data.frame`)\cr input data.
+#'
```

Comment:
Return is missing

Reply:
added now; also added for some other functions in utils.R